app/controllers/view-tree.js

```diff
@@ -38,6 +38,10 @@ export default Controller.extend({
   })),
 
   actions: {
+    clearSearchText() {
```

Comment:
It seems like we have a lot of different actions for clearing search. Could we perhaps name them all the same and maybe even have a base controller that has some common stuff to extend from?